From khali@linux-fr.org Sat Nov 26 19:24:50 2005
Date: Sat, 26 Nov 2005 20:54:10 +0100
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Cc: Laurent Riffard <laurent.riffard@free.fr>, Richard Purdie <richard@openedhand.com>
Subject: [PATCH 19/25] i2c: Drop i2c_driver.{owner,name}, 10 of 11
Message-Id: <20051126205410.4305279b.khali@linux-fr.org>
Content-Disposition: inline; filename=i2c-drop-driver-owner-and-name-10-arm.patch

We should use the i2c_driver.driver's .name and .owner fields
instead of the i2c_driver's ones.

This patch updates the drivers for arm arch.

Signed-off-by: Jean Delvare <khali@linux-fr.org>
CC: Laurent Riffard <laurent.riffard@free.fr>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 arch/arm/mach-pxa/akita-ioexp.c |    6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

--- gregkh-2.6.orig/arch/arm/mach-pxa/akita-ioexp.c
+++ gregkh-2.6/arch/arm/mach-pxa/akita-ioexp.c
@@ -124,8 +124,10 @@ static int max7310_detach_client(struct 
 }
 
 static struct i2c_driver max7310_i2c_driver = {
-	.owner		= THIS_MODULE,
-	.name		= "akita-max7310",
+	.driver = {
+		.owner	= THIS_MODULE,
+		.name	= "akita-max7310",
+	},
 	.id		= I2C_DRIVERID_AKITAIOEXP,
 	.attach_adapter	= max7310_attach_adapter,
 	.detach_client	= max7310_detach_client,
